client/views/cards/helpers.js

```diff
@@ -5,3 +5,14 @@ Template.cardLabelsPop.helpers({
       return _.contains(Cards.findOne(cardId).labelIds, this._id);
     }
 });
+
+Template.editLabelPop.helpers({
+    labels: function() {
+        var defaultLabels = ['green', 'yellow', 'orange', 'red', 'purple', 'blue'],
```

Comment:
Should we define these default elsewhere; because they seems identical to what we have in `collections/boards.js` so we could share this variable (DRY).